Sources/Tuvi.Core.Entities/EventArgs/LocalAIAgentEventArgs.cs

```diff
@@ -0,0 +1,15 @@
+﻿using System;
+
+namespace Tuvi.Core.Entities
+{
+    public class LocalAIAgentEventArgs : EventArgs
```

Comment:
Where is it used?

Reply:
All these code used here: https://github.com/ValeraFinebits/Eppie-App/tree/feat/local-ai